tony-cli/src/main/java/com/linkedin/tony/cli/ClusterSubmitter.java

```diff
@@ -33,12 +34,18 @@
  * --executes /Users/xxx/hadoop/li-tony_trunk/tony/src/test/resources/exit_0_check_env.py \
  * --python_binary_path python
  */
-public class ClusterSubmitter {
+public class ClusterSubmitter extends TonySubmitter {
   private static final Log LOG = LogFactory.getLog(ClusterSubmitter.class);
+  private TonyClient client;
 
-  private ClusterSubmitter() { }
+  private ClusterSubmitter() {
+    this.client = new TonyClient(new Configuration());
+  }
+  public ClusterSubmitter(TonyClient client) {
+    this.client = client;
+  }
 
-  public static void main(String[] args) throws URISyntaxException {
+  public int submit(String[] args) throws ParseException, URISyntaxException {
```

Comment:
Nit: Can we annotate these implementation methods with `@Override`. Same in NotebookSubmitter.java.